package.json

```diff
@@ -44,31 +44,31 @@
   },
   "homepage": "https://github.com/fastify/fastify-autoload#readme",
   "devDependencies": {
-    "@fastify/pre-commit": "^2.0.2",
-    "@fastify/url-data": "^5.0.0",
-    "@swc-node/register": "^1.5.1",
-    "@swc/core": "^1.2.129",
+    "@fastify/pre-commit": "^2.1.0",
+    "@fastify/url-data": "^5.4.0",
+    "@swc-node/register": "^1.8.0",
+    "@swc/core": "^1.4.2",
     "@swc/register": "^0.1.9",
-    "@types/jest": "^29.0.0",
-    "@types/node": "^20.1.0",
-    "@types/tap": "^15.0.5",
-    "fastify": "^4.0.0-rc.2",
-    "fastify-plugin": "^4.0.0",
-    "jest": "^28.1.3",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^20.11.24",
+    "@types/tap": "^15.0.11",
```

Comment:
why do we even have @types/tap? Can be removed.

Reply:
Because it is used in `typescript` testing.
https://github.com/fastify/fastify-autoload/blob/master/test/typescript/basic.ts#L1

Reply:
Pushed new version.

@Uzlopak  and @climba03003  I am 100% ok for your updating the packages file for no package lock. I just now that there is a combination or a single package causing this issue. I am just not sure which package makes my testing work.

But this is where I have problems. I not always sure what to do in this case. In all my apps, I keep the packages  them up to date frequently for security and just keeping the code up to date. I understand that this is a plugin need to be more universal for a bigger mass. So whatever changes you think, I will accept and then test them with my code.